xcodeproj/internal/bazel_integration_files/bazel_build.sh

```diff
@@ -93,11 +87,21 @@ bazel_cmd=(
   USER="$USER"
   "$BAZEL_PATH"
   "${bazelrcs[@]}"
-  ${output_base:+--output_base "$output_base"}
 )
 
 # Determine Bazel output_path
 
+if [ "$ACTION" == "indexbuild" ]; then
+  # We use a different output base for Index Build to prevent normal builds and
+  # indexing waiting on bazel locks from the other. We nest it inside of the
+  # normal `bazel-out` directory so that it's cleaned up when running
+  # `bazel clean`.
+
+  bazel_cmd+=(
+    --output_base "$BAZEL_OUT/_rules_xcodeproj/indexbuild_output_base"
```

Comment:
Can I request a small change? Can we put the index build output base directly under the main output base instead of under `bazel-out`? I think it's unlikely people want to clean the index store when they run `bazel clean`. Xcode also doesn't clean up the project's `DataStore` directory when you do Cmd + Shift + K. Nesting it in the main output base ensures that a `bazel clean --expunge` from the main build will clean it up.

Reply:
Yeah, sounds good, I'll make that change.